homeassistant/components/ecobee/config_flow.py

```diff
@@ -0,0 +1,173 @@
+"""Config flow to configure ecobee."""
+import voluptuous as vol
```

Comment:
Please sort imports and put 3rd party import group between standard library and homeassistant import groups.

Reply:
Is this the case for all imports from pyecobee? Should all pyecobee imports be imported at the top of the various files? I know it used to be the case that 3rd party imports needed to be inside the functions that used them.

Reply:
Yes all 3rd party imports including pyecobee should be put at the top. After we introduced manifest.json we can import everything at the module level.